contracts/contracts/strategies/AaveStrategy.sol

```diff
@@ -25,13 +25,34 @@ contract AaveStrategy is InitializableAbstractStrategy {
         onlyVault
         nonReentrant
     {
-        require(_amount > 0, "Must deposit something");
+        _deposit(_asset, _amount);
+    }
 
+    /**
+     * @dev Deposit asset into Aave
+     * @param _asset Address of asset to deposit
+     * @param _amount Amount of asset to deposit
+     * @return amountDeposited Amount of asset that was deposited
+     */
+    function _deposit(address _asset, uint256 _amount) internal {
```

Comment:
Since the Aave and Compound strategies are being modified, we should add them to the deploy script?